media-tv/mythtv/mythtv-34.0.ebuild

```diff
@@ -0,0 +1,376 @@
+# Copyright 1999-2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+DISABLE_AUTOFORMATTING="yes"
+PYTHON_COMPAT=( python3_{10..12} )
+
+inherit edo flag-o-matic java-pkg-opt-2 python-any-r1
+inherit qmake-utils readme.gentoo-r1 systemd toolchain-funcs user-info
+
+DESCRIPTION="Open Source DVR and media center hub"
+HOMEPAGE="https://www.mythtv.org https://github.com/MythTV/mythtv"
+if [[ ${PV} == *_p* ]] ; then
+	MY_COMMIT="5824c588db24b4e71a7d94e829e6419f71089297"
+	SRC_URI="https://github.com/MythTV/mythtv/archive/${MY_COMMIT}.tar.gz -> ${P}.tar.gz"
+	# mythtv and mythplugins are separate builds in the github MythTV project
+	S="${WORKDIR}/mythtv-${MY_COMMIT}/mythtv"
+else
+	SRC_URI="https://github.com/MythTV/mythtv/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+	# mythtv and mythplugins are separate builds in the github mythtv project
+	S="${WORKDIR}/${P}/mythtv"
+fi
+
+LICENSE="GPL-2+"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+
+IUSE_INPUT_DEVICES="input_devices_joystick"
+IUSE_VIDEO_CAPTURE_DEVICES="v4l ieee1394 hdhomerun vbox ceton"
+IUSE="alsa asi autostart cdda cdr cec cpu_flags_ppc_altivec debug dvd dvb exif fftw jack java"
+IUSE+=" +lame lcd libass lirc nvdec +opengl oss perl pulseaudio python raw systemd vaapi vdpau vpx"
+IUSE+=" +wrapper x264 x265 +xml xmltv +xvid +X zeroconf"
+IUSE+=" ${IUSE_INPUT_DEVICES} ${IUSE_VIDEO_CAPTURE_DEVICES}"
+REQUIRED_USE="
+	cdr? ( cdda )
+"
+
+RDEPEND="
+	acct-user/mythtv
+	dev-libs/glib:2
+	dev-libs/lzo:2
+	dev-libs/libzip:=
+	dev-qt/qtbase:6[dbus,gui,mysql,network,sql,xml,widgets]
+	media-fonts/corefonts
+	media-fonts/dejavu
+	media-fonts/liberation-fonts
+	media-fonts/tex-gyre
+	media-gfx/exiv2:=
+	media-libs/freetype:2
+	media-libs/libbluray:=[java?]
+	media-libs/libsamplerate
+	media-libs/libsoundtouch:=
+	media-libs/taglib:=
+	sys-libs/zlib
+	alsa? ( media-libs/alsa-lib )
+	autostart? (
+		net-dialup/mingetty
+		x11-apps/xset
+		x11-wm/evilwm
+	)
+	cec? ( dev-libs/libcec )
+	dvd? (
+		dev-libs/libcdio:=
+		media-libs/libdvdcss
+		sys-fs/udisks:2
+	)
+	fftw? ( sci-libs/fftw:3.0=[threads] )
+	hdhomerun? ( media-libs/libhdhomerun )
+	ieee1394? (
+		media-libs/libiec61883
+		sys-libs/libavc1394
+		sys-libs/libraw1394
+	)
+	jack? ( virtual/jack )
+	java? ( >=virtual/jdk-1.8 )
```

Comment:
jdk in RDEPEND looks confusing.

Reply:
First, rename RDEPEND => COMMON_DEPEND, then

DEPEND="${COMMON_DEPEND}"
	java? ( >=virtual/jdk-1.8:* )
}

RDEPEND="${COMMON_DEPEND}"
	java? ( >=virtual/jre-1.8:* )
}